tests/test-obsidian.el

```diff
@@ -2,35 +2,43 @@
 (require 'obsidian)
 (require 'buttercup)
 
-(defvar obsidian--test-dir "./tests/test_vault")
+;; Using a relative path for obsidian-change-vault/obsidian-change-vault will
+;; result in different values for obsidian-directory depending on the directory
+;; of the most recently visited file
+(defvar obsidian--test-dir (expand-file-name "./tests/test_vault"))
```

Comment:
Using a relative path here was causing issues when running tests after other tests that visited files because the relative location would then be the location of the opened note and not the vault root.